index.js

```diff
@@ -0,0 +1,86 @@
+const inquirer = require("inquirer");
```

Comment:
Nice job importing external modules and packages at the top of your file, before imports of code you have written!